update-rules.json

```diff
@@ -829,5 +828,286 @@
       }
     },
     "version_extractor": "^[0-9]{2}"
+  },
+    "winscp": {
+     "url": "https://winscp.net/eng/download.php",
+     "updater": {
+      "x86":{
+        "rule_type": "css-html-version",
+        "selector": "#content > h3:nth-child(8)",
```

Comment:
using "nth-child" is very risky as most changes to the page will break the rule. Better use regexp to get a surer match on the page, such as a section title on the page with the version number (e.g. WinSCP 5.9.3)